proto/regen/data/v1/tx.proto

```diff
@@ -34,11 +34,11 @@ service Msg {
   // - the blockchain transaction envelope provides built-in replay protection
   //   and timestamping
   //
-  // SignData implicitly calls AnchorData if the data was not already anchored.
+  // Attest implicitly calls Anchor if the data was not already anchored.
   //
-  // SignData can be called multiple times for the same content hash with different
-  // signers and those signers will be appended to the list of signers.
-  rpc SignData(MsgSignData) returns (MsgSignDataResponse);
+  // Attest can be called multiple times for the same content hash with different
+  // attestors and those attestors will be appended to the list of attestors.
```

Comment:
What happens if the same attestor attests same data multiple times? Is it overwritten?

Reply:
In the current implementation, we don't do anything if an attestor attempts to attest to the same data. The original attestation with the original timestamp is not overwritten and no new attestation is added. I can add a note here.